modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -0,0 +1,127 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/authz"
+
+	channeltypes "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
+	host "github.com/cosmos/ibc-go/v7/modules/core/24-host"
+)
+
+const gasCostPerIteration = uint64(10)
```

Comment:
nit, ideally this should be either:

a. a private field on the `TransferAuthorization` that is defined on instantiation, or
b. the value from `ctx.KVGasConfig.IterNextCostFlat`